.drone.yml

```
@@ -2334,34 +2252,34 @@ trigger:
     - pull_request
     - push
 
----
-kind: pipeline
-name: memcache-redis-cluster
-
-steps:
-- name: submodules
-  image: docker:git
-  commands:
-    - git submodule update --init
-- name: memcache-redis-cluster
-  image: nextcloudci/php7.1:php7.1-16
-  commands:
-    - phpenmod xdebug
-    - sleep 20
-    - ./autotest.sh sqlite tests/lib/Memcache/RedisTest.php
-    - wget https://codecov.io/bash -O codecov.sh
-    - sh -c "if [ '$DRONE_BUILD_EVENT' = 'pull_request' ]; then bash codecov.sh -B $DRONE_BRANCH -C $DRONE_COMMIT -P $DRONE_PULL_REQUEST -t 117641e2-a9e8-4b7b-984b-ae872d9b05f5 -f tests/autotest-clover-sqlite.xml; fi"
-    - sh -c "if [ '$DRONE_BUILD_EVENT' != 'pull_request' ]; then bash codecov.sh -B $DRONE_BRANCH -C $DRONE_COMMIT -t 117641e2-a9e8-4b7b-984b-ae872d9b05f5 -f tests/autotest-clover-sqlite.xml; fi"
-
-services:
-- name: cache-cluster
-  image: morrisjobke/redis-cluster
-
-trigger:
-  branch:
-    - master
-    - stable*
-  event:
-    - pull_request
-    - push
+#---
```

Any specific reason why that is disabled? :wink:

---

It fails hard on 7.2
But it also alrady failed on 7.1 but there it was just ignored by phpunit for some reason.

---

Ok then fine by me